docs/design/Policy Provider refactor (deprecated).md

```diff
@@ -21,9 +21,11 @@ Add a OverallVerifySuccess method to the interface. This will allow the policy p
 ```
 type VerifierResult struct {
 	Subject       string           `json:"subject,omitempty"`
-	IsSuccess     bool             `json:"isSuccess,omitempty"`
+	IsSuccess     bool             `json:"isSuccess"`
```

Comment:
I think we shouldn't update this document. It's an old design document and should stay consistent with its time.

Reply:
Well noted.